From 4f8bfc00d9f46eba403e15a0afdf47c708e613c4 Mon Sep 17 00:00:00 2001
From: Drew Moseley <drew.moseley@northern.tech>
Date: Wed, 11 Jul 2018 21:25:49 +0000
Subject: [PATCH] Add back some kernel defines that have been removed upstream.

This is a hack to get it past the build error and is not a proper fix.

Signed-off-by: Drew Moseley <drew.moseley@northern.tech>
---
 .../hal/os/linux/kernel/gc_hal_kernel_os.c                        | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/kernel-module-imx-gpu-viv-src/hal/os/linux/kernel/gc_hal_kernel_os.c b/kernel-module-imx-gpu-viv-src/hal/os/linux/kernel/gc_hal_kernel_os.c
index 090669f..e34fbe6 100644
--- a/kernel-module-imx-gpu-viv-src/hal/os/linux/kernel/gc_hal_kernel_os.c
+++ b/kernel-module-imx-gpu-viv-src/hal/os/linux/kernel/gc_hal_kernel_os.c
@@ -5969,6 +5969,14 @@ OnError:
     return status;
 }
 
+#ifdef __KERNEL__
+#define __SI_RT		(5 << 16)
+#define __SI_CODE(T,N)	((T) | ((N) & 0xffff))
+#else /* __KERNEL__ */
+#define __SI_RT		0
+#define __SI_CODE(T,N)	(N)
+#endif /* __KERNEL__ */
+
 #if gcdENABLE_VG
 gceSTATUS
 gckOS_SetSignalVG(
-- 
2.7.4

